README.md

```diff
@@ -123,6 +126,18 @@ Open a [pull request][pulls] when you are ready to contribute.
 If you are planning to contribute documentation, please check
```

Comment:
Probably super minor and comes down to personal preference, but I personally really don't like this way of writing Markdown 😹

My view on this: if one wants to have line-wrapping after a certain number of characters for local editing, everyone can do this in their editors the way they want. Doing the wrapping with actual line breaks _(which don't have any effect on the rendered HTML output)_ is enforcing a wrapping-style on every future developer.
In my opinion, this makes it a bit harder for future developers... What do you think?

This also applied to the following paragraphs.

Reply:
I agree! I didn't want to change it here to keep the diff clean, but once the other changes are in, I think we can simply add a commit on top removing the superfluous linebreaks.